fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -7,15 +7,17 @@ class SwiftFunction
     attr_accessor :param_default_values
     attr_accessor :param_optionality_values
     attr_accessor :param_type_overrides
+    attr_accessor :param_is_strings
     attr_accessor :reserved_words
     attr_accessor :default_values_to_ignore
 
-    def initialize(action_name: nil, keys: nil, key_descriptions: nil, key_default_values: nil, key_optionality_values: nil, key_type_overrides: nil, return_type: nil)
+    def initialize(action_name: nil, keys: nil, key_descriptions: nil, key_default_values: nil, key_optionality_values: nil, key_type_overrides: nil, key_is_strings: nil, return_type: nil)
```

Comment:
There's plenty of logic behind this action. Wish we had some unit tests 😅 Unfortunately dunno if that's possible, not very familiar with the way `fastlane` generates `Swift` files.